query-engine/nodejs-drivers/src/queryable.rs

```diff
@@ -35,28 +35,21 @@ impl std::fmt::Debug for Queryable {
 #[async_trait]
 impl QuaintQueryable for Queryable {
     /// Execute the given query.
-    async fn query(&self, q: Query<'_>) -> quaint::Result<quaint::prelude::ResultSet> {
+    async fn query(&self, q: QuaintQuery<'_>) -> quaint::Result<quaint::prelude::ResultSet> {
         let (sql, params) = visitor::Mysql::build(q)?;
         println!("JSQueryable::query()");
         self.query_raw(&sql, &params).await
     }
 
     /// Execute a query given as SQL, interpolating the given parameters.
     async fn query_raw(&self, sql: &str, params: &[Value<'_>]) -> quaint::Result<quaint::prelude::ResultSet> {
-        println!("JSQueryable::query_raw({}, {:?})", &sql, params);
+        println!("JSQueryable::query_raw()");
 
         // Note: we ignore the parameters for now.
         // Todo: convert napi::Error to quaint::error::Error.
-        let result_set = self.driver.query_raw(sql.to_string()).await.unwrap();
+        let result_set = self.driver.query_raw((sql, params).into()).await.unwrap();
```

Comment:
nitpick: I always prefer `Query::from((sql, params))` which I find a lot more readable usually as it doesn't hide the type you're transforming it into. We've got ton of `.into()` usage in the codebase so this is more of a FYI than a requirement to change the code